lib/config/fileconf.go

```diff
@@ -2619,6 +2605,32 @@ func (j *JamfService) toJamfSpecV1() (*types.JamfSpecV1, error) {
 	return spec, nil
 }
 
+func (j *JamfService) credentials() (servicecfg.JamfCredentials, error) {
```

Comment:
nit: Return a pointer. It allows us to `return nil, err` instead of `return servicecfg.JamfCredentials{}, err`.

Suggestion: Rename?

```suggestion
func (j *JamfService) toJamfCredentials() (*servicecfg.JamfCredentials, error) {
```